.github/workflows/release.yml

```
@@ -0,0 +1,267 @@
+          else
+            PATCH=$((PATCH + 1))
+          fi
+          
+          NEW_VERSION="v$MAJOR.$MINOR.$PATCH"
+          echo "New Version: $NEW_VERSION"
+          echo "::set-output name=new_version::$NEW_VERSION"
+
+      # Get the milestone details
+      - name: Get Milestone Details
+        id: get_milestone
+        run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)